src/pages/collections/[slug].astro

```diff
@@ -1,18 +1,14 @@
 ---
 import { getFullRelativePath } from "utils/url-paths";
 import SEO from "src/components/seo/seo.astro";
-import "../../page-components/collections/collections.scss";
-import "../../styles/post-body.scss";
-import "../../styles/tabs.scss";
-import "../../styles/shiki.scss";
-import "../../styles/convertkit.scss";
 import Document from "src/layouts/document.astro";
-import { getImageSize } from "../../utils/get-image-size";
-import CollectionHeaderDefault from "src/page-components/collections/collection-header-default.astro";
-import CollectionTableOfContents from "src/page-components/collections/collection-table-of-contents.astro";
 import { ExtendedCollectionInfo } from "types/CollectionInfo";
 import { MarkdownInstance } from "astro";
 import { getPostsByCollection } from "utils/api";
+import CollectionPage from "src/page-components/collections/collection-page.astro";
+import CollectionChapter from "src/page-components/collections/collection-chapter.astro";
```

Comment:
Not sure if this was planned, but reminder to move these from `src/page-components -> src/views` if not :)

Reply:
Done!